docs/sinks/aerospike.md

```diff
@@ -44,15 +44,13 @@ sinks:
 
 Additionally, Dozer also allows you to denormalize the destination data in Aerospike, allowing for faster read operations.
 
-You can enable denormalization by adding the `denormalize` parameter to the sink configuration. The `denormalize` parameter is a list of denormalization rules to apply to the data before writing it to the destination. Each rule is a dictionary with the following parameters:
-
-| **Parameter Name** | **Type**             | **Description**                                                                                                                                                                                                                                                             |
-|--------------------|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `from_namespace`             | String               | The namespace for the source record.                                                                                                                                                                 |
-| `from_set`         | String               | The set within the namespace for the source record.                                                                                                                  |
-| `key`             | String | The key to use to join the source and destination records.                                                                                                                                                                              |
-| `columns`             | List | A list of columns to copy from the source record to the destination record.                                                                                                                                                                              |
+With denormalization, you can combine tables that were previously seperated. This can be useful when you have a table with a lot of reads and you want to avoid joins. In essence, it is like a left-join, so the below example is equivalent to the following SQL query:
```

Comment:
I think we don't need to describe why you would use denormalization. If someone uses the product, they know whether they want to denormalize or not.